modules/veteran_confirmation/app/controllers/veteran_confirmation/v0/veteran_status_controller.rb

```diff
@@ -43,6 +46,13 @@ def valid_ssn?(ssn)
         ssn.is_a?(String) && (all_digits?(ssn) || all_digits_with_hyphens?(ssn))
       end
 
+      def validate_gender
+        gender_options = %w(M F)
+        no_matching_option = params['gender'] && !gender_options.include?(params['gender'])
```

Comment:
Shall we upcase `params['gender']` in the cases of 'm' and 'f'?

Reply:
Good call. I'll add in the upcasing.